### ISSUE TITLE
Unknown event: status_idle\.Kernel when executing cells

### ISSUE THREAD

**Issue by hnisonoff** [millejoh/emacs-ipython-notebook#579]
Thank you very much for this package, it is really great. I recently updated my emacs packages, including ein and have been running into issues:

## Problem description
Every time I execute a cell I get the following message: 
ein: [info] Unknown event: status_idle\.Kernel
## System info:

```cl
("EIN system info"
 :emacs-version
 "GNU Emacs 26.1 (build 1, x86_64-apple-darwin14.5.0, NS appkit-1348.17 Version 10.10.5 (Build 14F2511))
 of 2018-05-30"
 :emacs-bzr-version nil
 :window-system ns
 :emacs-variant nil
 :os
 (:uname
  "Darwin Hunters-MacBook-Pro.local 18.7.0 Darwin Kernel Version 18.7.0: Thu Jun 20 18:42:21 PDT 2019; root:xnu-4903.270.47~4/RELEASE_X86_64 x86_64
"
  :lsb-release nil)
 :notebook
 "Name: notebook
Version: 5.7.8
Summary: A web-based notebook environment for interactive computing
Home-page: http://jupyter.org
Author: Jupyter Development Team
Author-email: jupyter@googlegroups.com
License: BSD
Location: /anaconda3/lib/python3.7/site-packages
Requires: nbconvert, ipykernel, terminado, jinja2, tornado, pyzmq, nbformat, prometheus-client, jupyter-core, traitlets, ipython-genutils, jupyter-client, Send2Trash
Required-by: widgetsnbextension, rise, jupyterlab, jupyterlab-server, jupyterlab-launcher, jupyter
"
 :ipython "7.4.0
"
 :image-types
 (png gif tiff jpeg xpm xbm pbm)
 :image-types-available
 (png gif tiff jpeg xpm xbm pbm)
 :request
 (:backend curl)
 :ein
 (:version "0.16.1"
	   :source-dir "/Users/hunternisonoff/.emacs.d/elpa/ein-0.16.1/")
 :lib
 ((:name "websocket"
	 :path "~/.emacs.d/elpa/ein-0.16.1/websocket.elc"
	 :featurep t
	 :version-var websocket-version
	 :version "1.9")
  (:name "auto-complete"
	 :path "~/.emacs.d/elpa/auto-complete-1.5.1/auto-complete.elc"
	 :featurep t
	 :version-var nil
	 :version nil)
  (:name "request"
	 :path "~/.emacs.d/elpa/request-0.3.1/request.elc"
	 :featurep t
	 :version-var request-version
	 :version "0.3.0")
  (:name "deferred"
	 :path "~/.emacs.d/elpa/deferred-0.5.1/deferred.elc"
	 :featurep t
	 :version-var deferred:version
	 :version "0.5.0")
  (:name "polymode"
	 :path "~/.emacs.d/elpa/polymode-20190714.2017/polymode.elc"
	 :featurep t
	 :version-var nil
	 :version nil)
  (:name "markdown-mode"
	 :path "~/.emacs.d/elpa/markdown-mode-20190802.2215/markdown-mode.elc"
	 :featurep t
	 :version-var markdown-mode-version
	 :version "2.4-dev")
  (:name "dash"
	 :path "~/.emacs.d/elpa/dash-2.16.0/dash.elc"
	 :featurep t
	 :version-var nil
	 :version nil)
  (:name "s"
	 :path "~/.emacs.d/elpa/s-1.12.0/s.elc"
	 :featurep t
	 :version-var nil
	 :version nil)
  (:name "skewer-mode"
	 :path "~/.emacs.d/elpa/skewer-mode-1.8.0/skewer-mode.elc"
	 :featurep t
	 :version-var nil
	 :version nil)))
```

**Comment by dickmao** [comment]
`melpa-stable` is something of a misnomer as it's susceptible to bitrot and library regressions.

Please package-install from `melpa` (`M-x ein:version` to ensure it looks something like "20180812.512").

**Comment by hnisonoff** [comment]
I am seeing "20190813.2156" on `melpa`. Is this correct?

**Comment by millejoh** [comment]
That looks correct - I've also made a stable release (I hope) based on that last commit. Current stable version is v0.16.2.
